src/site/content/en/blog/a11y-tips-for-web-dev/index.md

```diff
@@ -0,0 +1,557 @@
+for providing alternative text for that image, analogous to the `alt` attribute.
+
+### Do your components provide semantic information?
+
+Assistive technology conveys semantic information
+that is otherwise expressed to sighted users via visual cues
+such as formatting, cursor style, or position.
+Native elements have this semantic information built-in by the browser,
+but for custom components you need to use
+[ARIA](http://www.w3.org/WAI/PF/aria/) to add the information.
+
+As a rule of thumb, any component that listens to a mouse click or hover event
+should not only have some kind of keyboard event listener,
+but also an ARIA role and potentially ARIA states and attributes.
+
+For example, a custom `<fancy-slider>` UI component might take an ARIA role of slider,
+which has some related ARIA attributes: `aria-valuenow`, `aria-valuemin` and `aria-valuemax`.
+By binding these attributes to the relevant properties on your custom component,
+you can allow users of assistive technology to interact with the element,
+change its value, and even cause the element's visual presentation to change accordingly.
+
+<figure class="w-figure w-figure--center">
+  <img src="./slider.png" class="w-screenshot" alt="A screenshot of a slider.">
+  <figcaption class="w-figcaption">
+    A range slider component.
+  </figcaption>
+</figure>
+
+```html
+<fancy-slider role="slider" aria-valuemin="1" aria-valuemax="5" aria-valuenow="2.5"></fancy-slider>
+```
+
+### Can users understand everything without relying on color?
+
+Color shouldn't be used as the only means of conveying information,
+such as indicating a status, prompting the user for a response,
+or distinguishing a visual custom component.
+For example, if you create a `<fancy-map>` component that uses color
+to distinguish between heavy, moderate, and light traffic,
+an alternative means of distinguishing traffic levels should also be made available.
+One solution might be to hover over an element to display information in a tooltip.
+
+### Is there sufficient contrast between text and images and their background?
+
+Any text content displayed in your component should meet the
+[minimum WCAG AA-level contrast threshold](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast).
+Consider providing a high-contrast theme that meets the
+[higher AAA threshold](http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast7),
+and ensure that user agent style sheets can be applied
+if users require custom contrast or different colors.
+You can use this [Color Contrast Checker](http://webaim.org/resources/contrastchecker/)
+as an aid when designing your component.
+
+### Is the moving or flashing content in your components stoppable and safe?
+
+Content that moves, scrolls, or blinks for more than five seconds
+should be able to be paused, stopped or hidden.
+In general, try to flash no more than three times per second.
```

Comment:
@addyosmani, I thought people generally recommend avoiding flashing content altogether? My knowledge may be out of date though...

Reply:
Update guidance to avoid flashing content whenever possible.